test/profiler/test_execution_trace.py

```diff
@@ -343,6 +352,7 @@ def test_execution_trace_repeat_in_loop(self):
             assert found_root_node
         assert event_count == expected_loop_events
 
+    @onlyCPU
```

Comment:
Does the change bypass the CUDA test?

Reply:
Yes. This test case is originally targeted for CPU, so i added the decorator. The code change is redundant, i removed for now.